### PR TITLE
fix(obj): fix crash of lv_obj_move_foreground/background when the parent of obj is NULL

### PR DESCRIPTION
**Pull request by FASTSHIFT**
### Description of the feature or fix
```c
static void obj_test()
{
    lv_obj_t* obj = lv_obj_create(NULL);
    lv_obj_move_foreground(obj); // or lv_obj_move_background
}
```

### Checkpoints
- [x] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [x] Run `code-format.py` from the `scripts` folder. [astyle](http://astyle.sourceforge.net/install.html) needs to be installed.
- [ ] Update the documentation

### CODE REVIEW

File: src/core/lv_obj_tree.c [hunk 1]
```diff
@@ -191,14 +191,19 @@ void lv_obj_move_to_index(lv_obj_t * obj, int32_t index)
 {
     LV_ASSERT_OBJ(obj, MY_CLASS);
 
+    lv_obj_t * parent = lv_obj_get_parent(obj);
+
+    if(!parent) {
+        LV_LOG_ERROR("parent is NULL");
```
Comment (kisvegabor) [inline src/core/lv_obj_tree.c]:
```suggestion
        LV_LOG_WARN("parent is NULL");
```

Errors are used when LVGL halts.

File: src/lv_api_map.h [hunk 2]
```diff
@@ -55,6 +55,11 @@ static inline LV_ATTRIBUTE_TIMER_HANDLER uint32_t lv_task_handler(void)
 static inline void lv_obj_move_foreground(lv_obj_t * obj)
 {
     lv_obj_t * parent = lv_obj_get_parent(obj);
+    if(!parent) {
+        LV_LOG_ERROR("parent is NULL");
```
Comment (kisvegabor) [inline src/lv_api_map.h]: Same here

File: src/core/lv_obj_tree.c (hunk 1 above)
Comment (FASTSHIFT) [reply src/core/lv_obj_tree.c]: done

File: src/lv_api_map.h (hunk 2 above)
Comment (FASTSHIFT) [reply src/lv_api_map.h]: done

**Comment by FASTSHIFT** [comment]
Why does it involve changes to lv_conf_internal?

**Comment by kisvegabor** [comment]
> Why does it involve changes to lv_conf_internal?

Mysteriously the `lv_conf_internal.h` correctness check became broken. See #3585 

Anyway, it's unrelated, so I merge this.

**Pull request merged**